regression_and_census.R

```diff
@@ -24,36 +184,34 @@ doubleYScale(obj1, obj2, add.ylab2 = TRUE, use.style=FALSE )
 
 CA_PM25HOUR_plot_max <-
   ggplot(PM_LA, aes(week, PM25_wk_avg)) +
+  theme_bw() +
```

Comment:
For publication purposes, it generally looks better to have a blank background rather than the default grey grid that ggplot uses. You can test out different [ggplot themes](https://ggplot2.tidyverse.org/reference/ggtheme.html) to see which one you like.